templates/federated_reporting/import.sh

```diff
@@ -50,6 +50,8 @@ failed=0
 log "Setting up schemas for import"
 declare -a hostkeys
 for file in $dump_files; do
+  file_size=$(stat --format %s "$file")
+  log "Processing $file with size $file_size bytes"
```

Comment:
this might not really be needed since the puller says how big and how long it took to pull the dump files

```
  notice: Q: ".../bin/bash /opt/": [Tue May 24 23:18:20 CEST 2022] Pulling dumps from: 192.168.1.231
Q: ".../bin/bash /opt/": receiving incremental file list
Q: ".../bin/bash /opt/": SHA=e51c4479c586bd253df1d8665d49d71338cf00001cdc8bda5d2fd115e01c028f.sql.gz
Q: ".../bin/bash /opt/": sent 43 bytes  received 876,710 bytes  584,502.00 bytes/sec
Q: ".../bin/bash /opt/": total size is 876,326  speedup is 1.00
```

Only advantage I suppose it is on one line

```
Q: ".../bin/bash /opt/": [Tue May 24 23:18:21 CEST 2022] Processing /opt/cfengine/federation//superhub/import/SHA=e51c4479c586bd253df1d8665d49d71338cf00001cdc8bda5d2fd115e01c028f.sql.gz with size 876326 bytes
```